packages/design-system/src/new-ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,41 @@
+import cn from "clsx";
+import * as React from "react";
+import * as TooltipPrimitive from "@radix-ui/react-tooltip";
+
+const TooltipProvider = TooltipPrimitive.Provider;
+const TooltipRoot = TooltipPrimitive.Root;
+const TooltipTrigger = TooltipPrimitive.Trigger;
+const TooltipPortal = TooltipPrimitive.Portal;
+
+const TooltipContent = React.forwardRef<
+  React.ElementRef<typeof TooltipPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof TooltipPrimitive.Content>
+>(({ className, sideOffset = 4, ...props }, ref) => (
+  <TooltipPrimitive.Content
+    ref={ref}
+    sideOffset={sideOffset}
+    className={cn(
+      "bg-primary text-primary-foreground z-50 overflow-hidden text-xs shadow-md animate-in fade-in-0 zoom-in-95 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2",
+      className
+    )}
+    {...props}
+  >
+    {props.children}
+    <TooltipPrimitive.Arrow
+      className="tooltip-arrow"
+      offset={10}
+      width={18}
+      height={12}
+      style={{ fill: "white" }}
```

Comment:
Please use Tailwind CSS fill-*

Reply:
We should expose Arrow as a compound component. For example, what if the user want to change the color of the Tooltip, they can't change the arrow color by current API.